making-aeternity-wallet-transactions-on-android.md

```diff
@@ -0,0 +1,147 @@
+# Tutorials How to make wallet transactions on Android
+
+## Tutorial Overview
+This tutorial is mend for android developers who want to begin to build interesting applications on the aeternity blockchain.
+In this tutorial we will cover the following:
+- Wallet Creation
+- Checking Wallet Balance
+- Token transfer
+
+## Prerequisites
+- A familiarity with the kotlin programming language
+- Basic understanding of some blockchain terminologies
+
+## Setup And Configuration
+Create a new android studio project, change the language type to kotlin, set the minimum sdk version to 26(Android 8.0 oreo)
+Now, in your app level build.gradle file, add the code below just after the closing brace of the defaultConfig
+
+```gradle
+ compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+    packagingOptions {
+        exclude 'META-INF/DISCLAIMER'
+        exclude 'META-INF/io.netty.versions.properties'
+        exclude 'META-INF/INDEX.LIST'
+    }
+    packagingOptions {
+        exclude 'lib/x86_64/darwin/libscrypt.dylib'
+        exclude 'lib/x86_64/freebsd/libscrypt.so'
+        exclude 'lib/x86_64/linux/libscrypt.so'
+    }
+```
+
+and finally add this dependency to the dependency section of your project and click ``sync now``
```

Comment:
please add that `jcenter()` needs to be included in the repositories section of the main `build.gradle`:
```
repositories {
        jcenter()
}
```

I think this is standard but it is an important requirement.

Reply:
Thanks, have fixed that now